### DISCUSSION TITLE
Can't login with my github account

### DISCUSSION THREAD

**Discussion by Seybo** [vercel/community#5209] [Account Issue] [answered]
### Question

Dear Vercel Support,

I am encountering a critical issue accessing my project due to login problems with my GitHub (GH) account. When attempting to log in, I receive the error: `There is no Vercel account linked with this GitHub account`. This issue may be related to a recent change in my GH account's email address.

This problem is highly urgent as it is preventing access to my project. I have sent two emails detailing this issue to [registration@vercel.com](mailto:registration@vercel.com) but have not received any response. This lack of communication is even more concerning than the problem itself.

Could you please confirm the receipt of this message and provide guidance on resolving this login issue? Please let me know if an alternative contact method or additional information is required from my side.

Thank you for your prompt attention to this matter.

**Comment by morhi** [comment, marked as answer]
I got the same issue. I wrote an email too, but I also just got access by logging in using my GitHub email address, which somehow worked.

**Reply by saruaralam-swd** [reply]
> I got the same issue. I wrote an email too, but I also just got access by logging in using my GitHub email address, which somehow worked.

same problem. so could you help me?

**Reply by amyegan** [reply]
You can log in using the email address from your GitHub account. If you're not sure which email address is associated with your account, you can check your email inbox to see where past Vercel notifications were sent. That will be the email address you can use to log in with email. 

https://vercel.com/login/email
